code/datums/status_effects/debuffs.dm

```diff
@@ -347,14 +347,24 @@
 	owner.update_canmove()
 	return ..()
 
-//STUN
+//STUN - prevents movement and actions, victim stays standing
 /datum/status_effect/incapacitating/stun
 	id = "stun"
 
-//IMMOBILIZED
+//IMMOBILIZED - prevents movement, victim can still stand and act
 /datum/status_effect/incapacitating/immobilized
 	id = "immobilized"
 
-//PARALYZED
+//PARALYZED - TODO: change all `Weaken()` procs to Paralyze if maintainers want it
```

Comment:
Thoughts on this? it would put us in line with TG but, it would be quite a bit of work and would cause conflicts with the current `Paralyze` status effect.